lights-out.js

```diff
@@ -16,4 +17,51 @@ class LightsOut {
             }
         }
     }
+
+    getNeighbors(row, col) {
+        const neighborArray = [];
+        
+        if(row < this.numRows - 1) {

```

Comment:
I notice there is no missing whitespace between `if` and `(row ...)`. Please insert spaces throughout to achieve consistency in your PR.

Also, I noticed the same issue with missing whitespace in other areas tangential to this PR. Do not fix those whitespace issues in this PR, but after I merge this PR, please submit a new PR fixing whitespace.

Reply:
Added whitespace after the `if` statements in `getNeighbors()`. I notice the same issue at other `if` statements and `for` loops. I assume these are the ones you want me to fix in another PR, therefore I will leave them for now.